packages/mobile/src/screens/explore-screen/SearchExploreScreen.tsx

```diff
@@ -0,0 +1,180 @@
+import { useCallback, useContext } from 'react'
+
+import { useExploreContent, useUsers } from '@audius/common/api'
+import { exploreMessages as messages } from '@audius/common/messages'
+import { ExploreCollectionsVariant } from '@audius/common/store'
+import { MOODS } from 'pages/search-page/moods'
+import { ImageBackground, ScrollView, Image } from 'react-native'
+
+import {
+  Flex,
+  IconSearch,
+  Paper,
+  Text,
+  TextInput,
+  TextInputSize
+} from '@audius/harmony-native'
+import imageSearchHeaderBackground from 'app/assets/images/imageSearchHeaderBackground2x.png'
+import { CollectionList } from 'app/components/collection-list'
+import { Screen, ScreenContent } from 'app/components/core'
+import { UserList } from 'app/components/user-list'
+// import { useNavigation } from 'app/hooks/useNavigation'
+import { useIsUSDCEnabled } from 'app/hooks/useIsUSDCEnabled'
+import { makeStyles } from 'app/styles'
+import { moodMap } from 'app/utils/moods'
+
+import { AppDrawerContext } from '../app-drawer-screen'
+import { AccountPictureHeader } from '../app-screen/AccountPictureHeader'
+import { SearchCategoriesAndFilters } from '../search-screen/SearchCategoriesAndFilters'
+
+import {
+  PREMIUM_TRACKS,
+  TRENDING_PLAYLISTS,
+  TRENDING_UNDERGROUND
+} from './collections'
+import { ColorTile } from './components/ColorTile'
+import { ExploreCarousel } from './components/ExploreCarousel'
+import { REMIXABLES } from './smartCollections'
+
+const tiles = [
+  TRENDING_PLAYLISTS,
+  TRENDING_UNDERGROUND,
+
+  PREMIUM_TRACKS,
+  REMIXABLES
+]
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
```

Comment:
heads up i want us to deprecate using makeStyles, and just do inline styles with `useTheme()`